docs/UserGuide.md

```diff
@@ -11,117 +11,222 @@
+Format: `complete "TASK" by/TIME on/DATE [t/TAG...]`  
+#### Adds a floating task to Mastermind  
+Format: `do "TASK" [t/TAG...]` 
+
  
-> Words in `UPPER_CASE` are the parameters, items in `SQUARE_BRACKETS` are optional, 
-> items with `...` after them can have multiple instances. Order of parameters are fixed. 
-> 
-> Persons can have any number of tags (including 0)
+>* Words in `UPPER_CASE` are the parameters, items in `[SQUARE_BRACKETS]` are optional, 
+> items with `...` after them can have multiple instances.
+>* Tasks will be added to the categories (event, deadlines, floating task) according to the keywords (`add`, `complete`, `do`).
+>* DATE is in DDMM format.
+>* Parameters can be in any order.
+>* Separate different tags with ','.
+>* There are no limit to the number of tags a task can have (including 0).
+
+Examples: <br>
+* Adding an event
+    * `add "Dinner", at/1900 to 2000 on/1010 t/Date,meals`
+* Adding a task with deadline
+    * `complete "CS2010 PS10" by/1000 on/1110 t/Assignment`
+* Adding a floating task
+    * `do "Pay school fees"`
+
+### Listing all tasks in current tab: `list`
+Shows a list of all tasks in the To Do List.<br>
+Format: `list [TAB_NAME]`
+
+> TAB_NAME includes:
+> 1. Home 		<br>
+> 2. Tasks  	<br>
+> 3. Events 	<br>
+> 4. Deadlines 	<br>
+> 5. Archive 
+
+### Finding all tasks containing any keyword in their description: `find`
+Finds tasks whose description contain any of the given keywords.<br>
+Format: `find KEYWORD [MORE_KEYWORDS]`
+
+>* The search is not case sensitive.
+>* The order of the keywords does not matter.
+>* Tasks matching at least one keyword will be returned.
 
 Examples: 
-* `add John Doe p/98765432 e/johnd@gmail.com a/John street, block 123, #01-01`
-* `add Betsy Crowe p/1234567 e/betsycrowe@gmail.com a/Newgate Prison t/criminal t/friend`
+* `find Dinner`  
+  Returns `Dinner on 10/10/16 at 1900hrs (Date)`.
+* `find "cs2010 ps10"`  
+  Returns `CS2010 PS10 on 11 Oct by 1000hrs (Assignment)`.
+  
+### Finding all tasks containing any tag keyword in their description: `findtag`
+Finds tasks whose tag contain any of the given keywords.<br>
+Format: `findtag KEYWORD [MORE_KEYWORDS]`
+
+>* The search is not case sensitive
+>* The order of the tag keywords does not matter
+>* Tasks matching at least one tag keyword will be returned
 
-#### Listing all persons : `list`
-Shows a list of all persons in the address book.<br>
-Format: `list`
+Examples: 
+* `find date`  
+  Returns `Dinner on 10 Oct at 1900hrs (Date,meals)`.
+* `find "exam,assignment"`  
+  Returns `CS2010 PS10 on 11 Oct by 1000hrs (Assignment)`.
 
-#### Finding all persons containing any keyword in their name: `find`
-Finds persons whose names contain any of the given keywords.<br>
-Format: `find KEYWORD [MORE_KEYWORDS]`
+### Editing a task : `edit`
+Edits the task identified by the index number used in the last task listing.  
+Format: `edit INDEX [TASK] [at/TIME] [on/DATE] [t/TAGS...]`
```

Comment:
you have three commands for adding a task, each of different type. however, edit is just one.
does `edit` handle all three types of tasks? if yes, why not `add` do the same ;-)